Assets/Scripts/GameStats.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEngine;
+
+public class GameStats
+{
+    private string nameSettingFile = "gameStats.json";  // Name of the stats file.
```

Comment:
We will stick to tabs in our projects, we should be careful to check the format of the files when making a PR

Reply:
Ah! I used the basic configuration of Visual Studio, I will change that immediately. Sorry, it must have been horrible to read.